src/main/app/case/CaseApi.ts

```diff
@@ -71,16 +89,29 @@ export class CaseApi {
     }
   }
 
-  public async triggerEvent(caseId: string, userData: Partial<Case>, eventName: string): Promise<CaseWithId> {
+  public async saveUserData(caseId: string, userData: Partial<Case>, eventName = CITIZEN_UPDATE): Promise<CaseWithId> {
+    return this.triggerEvent(caseId, toApiFormat(userData), eventName);
+  }
+
+  public async triggerEvent(caseId: string, data: Partial<CaseData>, eventName: string): Promise<CaseWithId> {
     const tokenResponse = await this.axios.get(`/cases/${caseId}/event-triggers/${eventName}`);
     const token = tokenResponse.data.token;
     const event = { id: eventName };
-    const data = toApiFormat(userData);
 
     try {
-      const response = await this.axios.post(`/cases/${caseId}/events`, { event, data, event_token: token });
-
-      return { id: response.data.id, state: response.data.state, ...fromApiFormat(response.data.data) };
+      const { data: { id, state, data: caseData } = {} } = await this.axios.post(`/cases/${caseId}/events`, {
+        event,
+        data,
+        event_token: token,
+      });
+
+      return {
+        ...fromApiFormat(caseData),
+        id,
+        state,
+        applicationFeeOrderSummary: caseData.applicationFeeOrderSummary,
```

Comment:
just add them to the pass through

Reply:
I'll take a look at moving these into into `case.ts` and make sure that `toApiFormat` removes them if the user is sending the request.